### ISSUE TITLE
Eliminate usage of crypt library

### ISSUE THREAD

**Issue by ekos2001** [Jc2k/aiohomekit#12]
Because of the crypt library is not available on Windows, I'm receiving an error in HA
`2020-08-03 00:45:40:0100	700	2020-08-03 00:45:39 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading configuration flow for integration homekit_controller: The crypt module is not supported on Windows`

As I can see, it is used in one place only to generate a salt in the aiohomekit/aiohomekit/crypto/srp.py
`salt = crypt.mksalt(crypt.METHOD_SHA512)[3:]`

May be it can be replaced with different implementation?

**Comment by Jc2k** [comment]
`crypt` is built in to python so it's a real shame they don't have an implementation on Windows. I suggested an alternative for homekit_python (the synchronous/blocking library aiohomekit was based off of) [here](https://github.com/jlusiardi/homekit_python/issues/185#issuecomment-616344895). But I don't have Windows to test, and i've been struggling a bit for time.

Are you able to test the suggestion I made? Or even better submit a PR?
